resources/definitions/lotmaxx_sc60.def.json

```diff
@@ -0,0 +1,72 @@
+                [-50.7,-29.5],
+                [46.9,-29.5],
+                [49.9,16.8]
+            ]
+        },
+        "gantry_height": { "value": 29 },
+        "machine_heated_bed": {"value": true},
+        "machine_start_gcode":{
+          "default_value":"G28 ;Home\nG92 E0 ;Reset Extruder\nG1 Z4.0 F3000 ;Move Z Axis up\nG1 X10.1 Y20 Z0.28 F5000.0 ;Move to start position\nG1 X10.1 Y200.0 Z0.28 F1500.0 E15 ;Draw the first line\nG1 X10.4 Y200.0 Z0.28 F5000.0 ;Move to side a little\nG1 X10.4 Y20 Z0.28 F1500.0 E30 ;Draw the second line\nG92 E0 ;Reset Extruder\nG1 Z2.0 F3000 ;Move Z Axis up\n"
+        },
+        "machine_end_gcode":{
+          "default_value":"G91 ;Relative positionning\nG1 E-2 F2700 ;Retract a bit\nG1 E-2 Z0.2 F2400 ;Retract and raise Z\nG1 X5 Y5 F3000 ;Wipe out\nG1 Z10 ;Raise Z more\nG90 ;Absolute positionning\n\nG1 X0 Y{machine_depth} ;Present print\nM106 S0 ;Turn-off fan\nM104 S0 ;Turn-off hotend\nM140 S0 ;Turn-off bed\n\nM84 X Y E ;Disable all steppers but Z\n"
+        },
+        "acceleration_print":{"value":1000},
+        "acceleration_travel":{"value":1000},
+        "acceleration_travel_layer_0":{"value":1000.0},
+        "expand_skins_expand_distance":{"value":0.8},
+        "fill_outline_gaps":{"default_value":false},
+        "infill_sparse_density":{"value":15},
+        "meshfix_maximum_resolution":{"value":0.05},
+        "optimize_wall_printing_order":{"value":true},
+        "retract_at_layer_change":{"value":false},
+        "retraction_amount":{"value":4.5},
+        "roofing_layer_count":{"value":1},
+        "skin_preshrink":{"value":0.8},
+        "speed_layer_0":{"value":30},
+        "speed_print":{"value":45},
+        "speed_roofing":{"value":35},
+        "speed_topbottom":{"value":35},
+        "speed_travel":{"value":80},
+        "speed_wall_0":{"value":32},
+        "speed_wall_x":{"value":32},
+        "support_infill_rate":{"value":5},
+        "support_pattern":{"default_value":"lines"},
+        "support_use_towers":{"value":false},
+        "wall_overhang_speed_factor":{"value":50},
+        "z_seam_corner":{"default_value":"z_seam_corner_any"},
+        "z_seam_relative":{"value":true},
+        "z_seam_type":{"default_value":"sharpest_corner"},
+        "zig_zaggify_infill":{"value":true},
+        "adhesion_type":{"default_value":"skirt"},
+        "prime_tower_enable":{"value":true},
+        "prime_tower_position_x":{"value": 50},
+        "prime_tower_position_y":{"value": 50},
+        "prime_tower_min_volume":{"value": 30},
+        "switch_extruder_retraction_amount": {"value": 100},
+        "switch_extruder_retraction_speeds": {"value": 60}
+    },
+    "metadata": {
+        "visible": true,
+        "author": "lotmaxx.com",
+        "manufacturer": "Lotmaxx",
+        "platform": "lotmaxx_sc_10_20_platform.3mf",
+        "machine_extruder_trains": {
+            "0": "lotmaxx_sc60_extruder_0",
+            "1": "lotmaxx_sc60_extruder_right"
+        },
```

Comment:
The left extruder file is named lotmaxx_sc60_**extruder_0**.def.json while the right is named lotmaxx_sc60_**extruder_right**.def.json. Please make them consistent.